octobot.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: octobot
+  name: octobot
+spec:
+  ports:
+    - port: 5001
+  selector:
+    service: octobot
+  type: LoadBalancer
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    service: octobot
+  name: octobot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      service: octobot
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        service: octobot
+    spec:
+      containers:
+        - image: drakkarsoftware/octobot:stable
+          name: octobot
+          ports:
+            - containerPort: 5001
+              name: octobot
+          env:
+            - name: IS_CLOUD_ENV
+              value: "true"
+          volumeMounts:
+            - mountPath: /octobot/logs
+              name: octobot-logs
+            - mountPath: /octobot/backtesting
+              name: octobot-backtesting
+            - mountPath: /octobot/tentacles
+              name: octobot-tentacles
+            - mountPath: /octobot/user
+              name: octobot-user
+      restartPolicy: Always
+      volumes:
+        - name: octobot-logs
+          emptyDir: {}
```

Comment:
does that mean we not save logs for an execution to another ?

Reply:
Logs will also be retrieved with the console output of the containers so I don't think we need to store them.

Reply:
The console output has to be as detailed as the logfiles one for kub in this case 👍

Reply:
I'm not sure we want to store hundreds of megabytes per bot per day with the DEBUG level.

Reply:
It should not be that much, it's reduced by a lot now. For example right now I have a 1h runing bot with 31Ko logs. It's to be analysed if we need debug logs or not I think in cloud bots

Reply:
I think it would be easier to have the logs collected in the output of the console even if it means modifying the logs level when we are in the cloud. It will be much simpler rather than going to recover log files if needed